<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellEdit</class>
 <widget class="QMainWindow" name="SpellEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>439</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>439</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>439</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0" colspan="3">
        <widget class="QComboBox" name="school"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="name"/>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QComboBox" name="subschool"/>
       </item>
       <item row="9" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="notes">
         <property name="minimumSize">
          <size>
           <width>291</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>291</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>School</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="level"/>
       </item>
       <item row="9" column="0">
        <widget class="QSpinBox" name="prepared"/>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="cast"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Prepared</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Subschool</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cast</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="12">
        <widget class="QTextBrowser" name="description"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="preparedButton">
         <property name="text">
          <string>+ Per Day</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="castButton">
         <property name="text">
          <string>+ Used</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear Counts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="markButton">
         <property name="text">
          <string>Mark</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
